src/frontend/Komponenter/Behandling/Simulering/SimuleringTabellWrapper.tsx

```diff
@@ -56,7 +60,7 @@ const SimuleringTabellWrapper: React.FC<{
 
     const lagretSanksjonertVedtak =
         lagretVedtak?.resultatType === EBehandlingResultat.SANKSJONERE
-            ? (lagretVedtak as ISanksjonereVedtak)
+            ? (lagretVedtak as ISanksjonereVedtakForOvergangsstønad)
```

Comment:
Oppretter en egen oppgave på følge opp om vi trenger å vise sanksjon/håndtering av sanksjon for BT
https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-8239